playground/src/components/selectors/data_mode_select.tsx

```diff
@@ -1,14 +1,72 @@
 import type { DataMode } from "@/model/data_mode";
+import { useQueryClient } from "react-query";
+import useExercises from "@/hooks/playground/exercises";
+import useImportEvaluationData from "@/hooks/playground/evaluation_data/import";
+import useExportEvaluationData from "@/hooks/playground/evaluation_data/export";
+import useDeleteEvaluationData from "@/hooks/playground/evaluation_data/delete";
+
+function sanitizeDirectoryName(directory: string): string {
+  return directory.replace(/[^a-zA-Z0-9_-]/g, "");
+}
 
 export default function DataModeSelect({
   dataMode,
-  onChangeDataMode,
+  onChangeDataMode
 }: {
   dataMode: DataMode;
   onChangeDataMode: (dataMode: DataMode) => void;
 }) {
+  const queryClient = useQueryClient();
+  const evaluationDataKey = dataMode.startsWith("evaluation-") ? dataMode.slice("evaluation-".length) : undefined;
```

Comment:
Introduce a constant for the string literal?

```suggestion
  const evaluationKey = "evaluation-";
  const evaluationDataKey = dataMode.startsWith(evaluationKey) ? dataMode.slice(evaluationKey.length) : undefined;
```